app/Http/Controllers/TeamsController.php

```diff
@@ -118,6 +132,31 @@ public function show(string $id): Response
         return ext_view('teams.show', compact('team'));
     }
 
+    public function store(): Response
+    {
+        priv_check('TeamStore')->ensureCan();
+
+        $params = get_params(\Request::all(), 'team', [
+            'name',
+            'short_name',
+        ]);
+
+        $user = \Auth::user();
+        $team = (new Team([...$params, 'leader_id' => $user->getKey()]));
+        try {
+            \DB::transaction(function () use ($params, $team, $user) {
+                $team->saveOrExplode();
+                $team->members()->make(['user_id' => $user->getKey()])->saveOrExplode();
```

Comment:
I don't think this makes a difference over `create(...)` when there's no validation attached to the model 🤔

Reply:
there's still unique constraint...?